app/src/input_manager.c

```diff
@@ -498,9 +521,11 @@ input_manager_process_key(struct input_manager *im,
             case SDLK_n:
                 if (control && !repeat && down) {
                     if (shift) {
-                        collapse_notification_panel(controller);
-                    } else {
+                        collapse_panels(controller);
+                    } else if(repeatCount == 0) {
```

Comment:
It could also probably be done on double-click on mouse button 5 now :wink:

Reply:
Exactly! That PR is nearly ready ^^